### ISSUE TITLE
Error in rAI Toolbox 3rd Tutorial Example - Workflows

### ISSUE THREAD

**Issue by yumakemore** [mit-ll-responsible-ai/responsible-ai-toolbox#129]
While quickly trying the 3rd example at https://mit-ll-responsible-ai.github.io/responsible-ai-toolbox/tutorials/Building-Workflows.html , we got the following error (screenshot):
![image001](https://user-images.githubusercontent.com/34527859/215132979-13b031d2-d9a9-4b60-939a-4a5fc218b4a7.png)

**Comment by jgbos** [comment]
Thanks for reporting this.  I'll take a look and get back to you.  I hope to do a lot of clean up of docs and examples over the next few weeks. If you see any other issues let me know.

**Comment by jgbos** [comment]
Ouch.   This is due to a breaking update in [torchmetrics.Accuracy](https://torchmetrics.readthedocs.io/en/stable/classification/accuracy.html)!  Very unfortunate.  I believe you need replace all occurrences of

```python
Accuracy()
```
with 

```python
Accuracy(task="muliclass", num_classes=10)
```

**Comment by jgbos** [comment]
Looks like Lightning introduced another breaking change for calculating gradients in the perturbation at test time.  Replace the line in the `_step` method of `StandardModule`:

```python
data, adv_loss = self.perturber(
    model=self.model, data=data_orig, target=target
)
```
with

```python
inference_tensors = tr.is_inference_mode_enabled()
with tr.inference_mode(mode=False), tr.enable_grad():
    if inference_tensors:
        # we need to clone in order to support grad mode
        data_orig = data_orig.clone()
        target = target.clone()

    data, adv_loss = self.perturber(
        model=self.model, data=data_orig, target=target
    )
```

**Comment by jgbos** [comment]
@yumakemore I updated the docs.  Feel free to reopen if you still have issues.
